apps/sms/views/conversation/js/conversation.js

```diff
@@ -2480,26 +2483,23 @@ var ConversationView = {
     // force a number
     var id = +messageDOM.dataset.messageId;
     var iccId = messageDOM.dataset.iccId;
-
-    var request = MessageManager.retrieveMMS(id);
```

Comment:
> But now I think this be the role of MessagingClient, deciding to either use a worker or to load the worker script inside the window for a fast path.

Yep, I like this solution more - it gives us more granular control on what we want to replace if needed on contrary to the monolithic catch-all MessageManager.